client/src/components/Zip-code.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import FilledInput from '@material-ui/core/FilledInput';
+import FormControl from '@material-ui/core/FormControl';
+import FormHelperText from '@material-ui/core/FormHelperText';
+import Input from '@material-ui/core/Input';
+import InputLabel from '@material-ui/core/InputLabel';
+import OutlinedInput from '@material-ui/core/OutlinedInput';
+
+const styles = theme => ({
+    container: {
+        display: 'block',
+        marginLeft: 'auto',
+        marginRight: 'auto',
+        flexWrap: 'wrap',
+    },
+    formControl: {
+        margin: theme.spacing.unit,
+    },
+});
+
+class ComposedTextField extends React.Component {
+    constructor(props){
+        super(props)
+        this.state = {
+            name: ''
```

Comment:
small change but if I'm reading this correctly, I think it would make more sense for this.state to be 'zipcode' instead of name

Reply:
gotcha